lib/div-alg.gi

```diff
@@ -2166,7 +2166,7 @@ local m,A,B,n1;
 
 #A:=SimpleComponentOfGroupRingByCharacter(F,G,n);
 B:=Irr(G);
-if IsOrdinaryCharacter(n) then
+if IsCharacter(n) then
```

Comment:
This seems to me like a "fixup" - an adjustment of the previous commit. Alternatively, you could do that with
```
git add lib/div-alg.gi
git commit --amend
```
This will amend the last commit and incorporate there changes that you have just made. If you are working on a branch from which you have made a PR, and already pushed the previous version of an amended commit there, use `git push remote_name --force` or `git push remote_name -f` for short to overwrite that commit on GitHub. 

It's fine to use `--force` in this case, but never use it on a branch shared with others  without coordinating with them.